erica_app/erica/elster_xml/est_mapping.py

```diff
@@ -23,6 +23,8 @@
     'person_a_beh_grad': 'E0109708',
     'person_a_blind': 'E0109706',
     'person_a_gehbeh': 'E0109707',
+    
+    'telephone_number': 'E0100008',
```

Comment:
Could you perhaps place this between the person b and iban fields? (just so that it makes more sense in my head for the order of the steps^^

Reply:
Done!